go.mod

```diff
@@ -7,17 +7,15 @@ require (
 	github.com/chzyer/readline v0.0.0-20180603132655-2972be24d48e // indirect
 	github.com/chzyer/test v0.0.0-20180213035817-a1ea475d72b1 // indirect
 	github.com/ghodss/yaml v1.0.0
+	github.com/golangci/golangci-lint v1.30.0 // indirect
```

Comment:
I think this may have added several items to go.sum.

Reply:
I expect we can remove this line. I'm not sure how it was added. 
In some of the other projects we've found ways to get testing dependencies into a different go.mod file, but I don't think that's what is happening here.